### PR TITLE
chore: remove deprecated chains

### PR DESCRIPTION
**Pull request by vm**
WIP

### CODE REVIEW

File: src/hooks/useClientSideV3Trade.ts [hunk 1]
```diff
@@ -1,7 +1,5 @@
-import { Currency, CurrencyAmount, TradeType } from '@uniswap/sdk-core'
+import { Currency, CurrencyAmount, SupportedChainId, TradeType } from '@uniswap/sdk-core'
```
Comment (vm) [inline src/hooks/useClientSideV3Trade.ts:1]: fix

File: src/hooks/useStablecoinPrice.ts [hunk 2]
```diff
@@ -1,11 +1,10 @@
-import { Currency, CurrencyAmount, Price, Token, TradeType } from '@uniswap/sdk-core'
+import { Currency, CurrencyAmount, Price, SupportedChainId, Token, TradeType } from '@uniswap/sdk-core'
```
Comment (vm) [inline src/hooks/useStablecoinPrice.ts:1]: fix

File: src/state/governance/hooks.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ import { toUtf8String, Utf8ErrorFuncs, Utf8ErrorReason } from '@ethersproject/st
 import { t } from '@lingui/macro'
 import { abi as GOVERNANCE_ABI } from '@uniswap/governance/build/GovernorAlpha.json'
 import { abi as UNI_ABI } from '@uniswap/governance/build/Uni.json'
-import { CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { CurrencyAmount, SupportedChainId, Token } from '@uniswap/sdk-core'
```
Comment (vm) [inline src/state/governance/hooks.ts:11]: fix

File: src/components/FeeSelector/shared.tsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { Trans } from '@lingui/macro'
 import { FeeAmount } from '@uniswap/v3-sdk'
-import { ALL_SUPPORTED_CHAIN_IDS, SupportedChainId } from 'constants/chains'
+import { SupportedChainId } from 'constants/chains'
```
Comment (zzmp) [inline src/components/FeeSelector/shared.tsx:3]: surprised your linter didn't consolidate to one line

File: src/constants/networks.ts [hunk 5]
```diff
@@ -15,7 +15,7 @@ if (typeof INFURA_KEY === 'undefined') {
  *
  * These "Safe" URLs are listed first, followed by other fallback URLs, which are taken from chainlist.org.
  */
-export const FALLBACK_URLS: { [key in SupportedChainId]: string[] } = {
+export const FALLBACK_URLS = {
```
Comment (zzmp) [inline src/constants/networks.ts:18]: Why did you remove the typing?

File: src/constants/networks.ts [hunk 6]
```diff
@@ -93,21 +75,12 @@ export const FALLBACK_URLS: { [key in SupportedChainId]: string[] } = {
  * Known JSON-RPC endpoints.
  * These are the URLs used by the interface when there is not another available source of chain data.
  */
-export const RPC_URLS: { [key in SupportedChainId]: string[] } = {
+export const RPC_URLS = {
```
Comment (zzmp) [inline src/constants/networks.ts:78]: Why did you remove the typing?

File: src/constants/chains.ts [hunk 7]
```diff
@@ -3,13 +3,9 @@
  */
 export enum SupportedChainId {
```
Comment (zzmp) [inline src/constants/chains.ts:4]: Why doesn't this export from sdk-core?

File: src/components/FeeSelector/shared.tsx (hunk 4 above)
Comment (vm) [reply src/components/FeeSelector/shared.tsx:3]: i did this using find and replace in shell

**Review by zzmp — APPROVED** [review]

File: src/constants/chains.ts (hunk 7 above)
Comment (vm) [reply src/constants/chains.ts:5]: added comment
